package.json

```diff
@@ -35,56 +35,56 @@
     ]
   },
   "dependencies": {
-    "@angular/animations": "~13.0.0",
-    "@angular/common": "~13.0.0",
-    "@angular/compiler": "~13.0.0",
-    "@angular/core": "~13.0.0",
-    "@angular/forms": "~13.0.0",
-    "@angular/localize": "^13.0.0",
-    "@angular/platform-browser": "~13.0.0",
-    "@angular/platform-browser-dynamic": "~13.0.0",
-    "@angular/router": "~13.0.0",
+    "@angular/animations": "^14.0.0",
+    "@angular/common": "^14.0.0",
+    "@angular/compiler": "^14.0.0",
+    "@angular/core": "^14.0.0",
+    "@angular/forms": "^14.0.0",
+    "@angular/localize": "^14.0.0",
+    "@angular/platform-browser": "^14.0.0",
+    "@angular/platform-browser-dynamic": "^14.0.0",
+    "@angular/router": "^14.0.0",
     "@ng-bootstrap/ng-bootstrap": "^9.0.2",
```

Comment:
We should probably bump this to include any changes in the last few releases. I forget what this was used for, but after ng-bootstrap 12 it's Bootstrap 5 and I think they are still on Angular 13.

Was just noticing this when running yarn against your branch locally:

```
warning " > @ng-bootstrap/ng-bootstrap@9.1.3" has incorrect peer dependency "@angular/common@^11.0.0".
warning " > @ng-bootstrap/ng-bootstrap@9.1.3" has incorrect peer dependency "@angular/core@^11.0.0".
warning " > @ng-bootstrap/ng-bootstrap@9.1.3" has incorrect peer dependency "@angular/forms@^11.0.0".
warning " > @ng-bootstrap/ng-bootstrap@9.1.3" has incorrect peer dependency "@angular/localize@^11.0.0".
warning " > @ng-bootstrap/ng-bootstrap@9.1.3" has incorrect peer dependency "rxjs@^6.5.5".
```

Reply:
It looks like this is just used in the demo app https://github.com/pavankjadda/ng-select/pull/3/commits/cae10e89debf3d484b88cf442ababf9fae80e05e